packages/dicomImageLoader/src/imageLoader/decodeImageFrame.ts

```diff
@@ -14,9 +14,15 @@ function processDecodeTask(
   imageFrame: ImageFrame,
   transferSyntax: string,
   pixelData: ByteArray,
-  options,
+  srcOptions,
   decodeConfig: LoaderDecodeOptions
 ): Promise<ImageFrame> {
+  const options = { ...srcOptions };
+  // If a loader is specified, it can't be passed through
+  delete options.loader;
+  // streamData may contain larger data information and shouldn't be passed
+  delete options.streamingData;
```

Comment:
This is unusual and unexpected to me. Why? I know the documentation says it can't pass through here, but the reason (what happens if it passes) should be discussed as well.